docs/install/management-address.md

```diff
@@ -33,6 +55,34 @@ Example of output:
 {"kube-vip.io/hwaddr":"02:00:00:09:7f:3f","kube-vip.io/requestedIP":"10.84.102.31"}
 ```
 
+## Finding which node the VIP is on
+
+Using `kubectl` (either on your own machine with the Harvester kubeconfig, or SSH'd into any Harvester management node as root):
+
+```console
+kubectl get pod -n harvester-system -o custom-columns='NAME:.metadata.name,NODE:.spec.nodeName' | grep -E '^kube-vip|^NAME'
+```
+
+Example output:
+```console
+NAME                                                    NODE
+kube-vip-hxzfk                                          harvester-xzj76
+```
```

Comment:
This part might need to be more precise. The `kube-vip` Pods run on control-plane nodes, so if there are three Harvester management nodes, three Pods will be listed in the command output, and people still don't know which node the VIP address is on.

I'd suggest we could use the following instead:

```shell
kubectl -n kube-system get svc ingress-expose -o jsonpath='{.metadata.annotations.kube-vip\.io/vipHost}'
```